index.html

```diff
@@ -50,6 +50,92 @@ <h2 class="iam">I'm Sotfware Developer</h2>
       </ul>
     </div>
   </section>
+
+  <section class="work-seccion">
+    <div class="work-one">
+      <h1 class="title-work-one">My Recent Works</h1>
+      <img class="image-recent-work" src="./images/Placeholder@2x.png" alt="This is a meaningful image description">
+      <h2 class="multi-post">Multi-post stories</h2>
+      <p class="text-work-one">A daily selection of privately personalized reads; no accounts or sign-ups required. has
+        been the industry's
+        standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.
+      </p>g
+      <ul class="tag-recent-work">
+        <li> CSS</li>
+        <li>HTML</li>
+        <li>Bootstrap</li>
+        <li>Ruby</li>
+      </ul>
+      <button type="button" class="button-see-project">See project</button>
+    </div>
```

Comment:
- Good job working on this milestone, you are almost there 👏🏿 

Figma Design          |  Your design     
:-------------------------:|:-------------------------:
![Screenshot_50](https://user-images.githubusercontent.com/69638013/164955869-42ae75c9-db3b-43bc-978f-4dccd2102b22.png)  |  ![Screenshot_48](https://user-images.githubusercontent.com/69638013/164955825-da5d2efe-c89c-417e-9b11-58fa83af3996.png)

- [ ] Kindly change the my recent works header title to any header tag from h2 - h6, you currently have another h1 tag on your portfolio, this will make your website more semantic 👍🏿 
- [ ] Kindly adjust the image for the card to have same width as Figma design
- [ ] Please add the horizontal line after the header tag 
- [ ] Kindly add the right border outline to the li tags 
- [ ] Please  add more padding to the button, remove border outline and add button interactions to your button to look like the Figma design